### ISSUE TITLE
panic: interface conversion: interface {} is Config, not *Config

### ISSUE THREAD

**Issue by azr** [hashicorp/packer#8525]
We recently merged a fix for this but it should show something like this:

```
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: panic: interface conversion: interface {} is ovf.Config, not *ovf.Config
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin:
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: goroutine 48 [running]:
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: github.com/hashicorp/packer/builder/virtualbox/ovf.(*StepImport).Cleanup(0xc00025b580, 0x833a280, 0xc00053c120)
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: 	/Users/sylviamoss/go/src/github.com/hashicorp/packer/builder/virtualbox/ovf/step_import.go:45 +0x329
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: github.com/hashicorp/packer/helper/multistep.(*BasicRunner).Run(0xc00053c270, 0x835cfc0, 0xc00025b540, 0x833a280, 0xc00053c120)
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: 	/Users/sylviamoss/go/src/github.com/hashicorp/packer/helper/multistep/basic_runner.go:79 +0x2c6
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: github.com/hashicorp/packer/builder/virtualbox/ovf.(*Builder).Run(0xc0001a4580, 0x835cfc0, 0xc00025b540, 0x8377fc0, 0xc00053c0f0, 0x82e9980, 0xc000534d40, 0x203000, 0x203000, 0x203000, ...)
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: 	/Users/sylviamoss/go/src/github.com/hashicorp/packer/builder/virtualbox/ovf/builder.go:166 +0x12bc
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: github.com/hashicorp/packer/packer/rpc.(*BuilderServer).Run(0xc000240100, 0x1, 0xc0000583f0, 0x0, 0x0)
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: 	/Users/sylviamoss/go/src/github.com/hashicorp/packer/packer/rpc/builder.go:117 +0x27c
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: reflect.Value.call(0xc0000d4300, 0xc0004320f0, 0x13, 0x7c72011, 0x4, 0xc0000a0f18, 0x3, 0x3, 0xc000510678, 0x4006e3c, ...)
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: 	/usr/local/go/src/reflect/value.go:460 +0x5f6
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: reflect.Value.Call(0xc0000d4300, 0xc0004320f0, 0x13, 0xc000510718, 0x3, 0x3, 0xc00043b260, 0xc000569b00, 0xc000569b88)
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: 	/usr/local/go/src/reflect/value.go:321 +0xb4
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: net/rpc.(*service).call(0xc000240140, 0xc000448190, 0xc0000583b0, 0xc0000583c0, 0xc0001a8200, 0xc000444120, 0x7049c80, 0xc0000583bc, 0x18a, 0x6f887e0, ...)
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: 	/usr/local/go/src/net/rpc/server.go:377 +0x16f
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: created by net/rpc.(*Server).ServeCodec
2019/12/19 12:43:58 packer-builder-virtualbox-ovf plugin: 	/usr/local/go/src/net/rpc/server.go:474 +0x42b
```

This bug was fixed in #8513 

It happens in the following builders : 

* cloudstack
* hyperv/iso
* hyperv/vmcx
* linode
* lxc
* lxd
* oneandone
* oracle/classic
* oracle/oci
* osc/bsuvolume
* parallels/pvm
* profitbricks
* scaleway
* vagrant
* virtualbox/ovf

**Comment by azr** [comment]
The nightly build of packer contains the fix: https://github.com/hashicorp/packer/releases/tag/nightly

**Comment by SwampDragons** [comment]
This affects enough builders that today I'll be pushing a fast-follow 1.5.1 release with the fix.

**Comment by SwampDragons** [comment]
Closed by the 1.5.1 release.
